back/api/management/commands/fixturize.py

```diff
@@ -20,6 +21,7 @@ def handle(self, *args, **options):
 
         content_type = ContentType.objects.get_for_model(Order)
         extract_group = Group.objects.create(name='extract')
+        Group.objects.create(name='internal')
```

Comment:
I do not get where the creation of this goes? It looks like you that you create something, but it is not associated to a variable?

Reply:
.create() instanciates it directly in database. I could have done a migration instead of doing this "fixturize.py". In this case, I just want to be sure there will always be a group called `internal` in my databases either they're production or testing db.

Reply:
Ok, got it, thanks!